src/utils/analytics.ts

```diff
@@ -1,63 +1,17 @@
-import Analytics from 'analytics-node';
-import { createHash } from 'crypto';
 import { get, split, truncate } from 'lodash';
-import { nanoid } from 'nanoid';
 import type { UIEvent } from 'react';
 
-import { networkId } from './config';
+import * as RudderAnalytics from './rudder-analytics';
+import * as Segment from './segment-analytics';
 
-let segment: Analytics | null = null;
-let anonymousUserId = '';
-let hashId = '';
-let anonymousUserIdCreatedAt = '';
-
-export function setAccountIdHash(accountId: string) {
-  const hash = createHash('sha512');
-  hash.update(accountId);
-  hashId = hash.digest('hex');
-  localStorage.setItem('hashId', hashId);
-}
-
-function getAnonymousId() {
-  if (anonymousUserId) {
-    return anonymousUserId;
-  }
-
-  const storageId = localStorage.getItem('anonymousUserId');
-  anonymousUserIdCreatedAt = localStorage.getItem('anonymousUserIdCreatedAt') || '';
-
-  if (storageId) {
-    anonymousUserId = storageId;
-  } else {
-    anonymousUserId = nanoid();
-    anonymousUserIdCreatedAt = new Date().toUTCString();
-    localStorage.setItem('anonymousUserId', anonymousUserId);
-    localStorage.setItem('anonymousUserIdCreatedAt', anonymousUserIdCreatedAt);
-  }
-
-  return anonymousUserId;
+export async function init() {
```

Comment:
this file calls both 'Segment' methods as well as 'Rudderstack' methods for event tracking. 

After the migration period, we will remove Segment.